docroot/themes/custom/yukonca_glider/src/js/theme.js

```diff
@@ -39,4 +39,23 @@
       });
     },
   };
+
+  $(document).ready(function () {
```

Comment:
Please use [Behaviors](https://www.drupal.org/node/756722#behaviors) and [once](https://www.drupal.org/node/756722#jquery-once) for custom javascript

Reply:
Please check now.